PSReadLine/Completion.cs

```diff
@@ -532,8 +540,18 @@ public void DrawMenu(Menu previousMenu)
                     }
                 }
 
-                console.RestoreCursor();
-                console.CursorVisible = true;
+                if (menuSelect)
+                {
+                    console.RestoreCursor();
+                    console.CursorVisible = true;
+                }
+                else
+                {
+                    // Update the cursor coordinates after showing the menu.
```

Comment:
Is this block necessary? `InvokePrompt` is called immediately after the call site that passes `menuSelect: false` and `InvokePrompt` is correctly updating these fields.

I tested briefly with these commented out and didn't see any problems.